### ISSUE TITLE
NVDA doesn't read whole content when rich by arrow keys

### ISSUE THREAD

**Issue by Yegorich555** [nvaccess/nvda#11279]
### Steps to reproduce:
Just create ul>li>a with 16 items
```html
<ul>
   <li><a href="#">item 1</a></li>
   <li><a href="#">item 2</a></li>
   <li><a href="#">item 3</a></li>
   <li><a href="#">item 4</a></li>
   <li><a href="#">item 5</a></li>
   <li><a href="#">item 6</a></li>
   <li><a href="#">item 7</a></li>
   <li><a href="#">item 8</a></li>
   <li><a href="#">item 9</a></li>
   <li><a href="#">item 10</a></li>
   <li><a href="#">item 11</a></li>
   <li><a href="#">item 12</a></li>
   <li><a href="#">item 13</a></li>
   <li><a href="#">item 14</a></li>
   <li><a href="#">item 15</a></li>
   <li><a href="#">item 16</a></li>
</ul>
```

### Actual behavior:
When you reach list with arrow keys NVDA reads 15 items. After that if you continue to press arrow-key NVDA continues read 'item 16' with setting focus on in.

### Expected behavior:
NVDA should read the whole content

### Reproducible link:
https://codepen.io/yegorich_555/pen/jOWyLde 

###Browser
Chrome 83+

I'm not sure is it bug or feature. I guess NVDA has length-reading-at-once restriction, is it changeable or not?

**Comment by lukaszgo1** [comment]
@Yegorich555 In what browser have you tested this? It works correctly for me in Firefox Nightly 79.

**Comment by XLTechie** [comment]
I believe it is the browser doing some wrapping here.

It is probably dependent on screen width.

If I look at this in Brave, I get the behavior you described (I think, anyway): 
on one line I get 15 of the items, and have to arrow to the next line to get the 
16th.

However if I look at it in Firefox, I get only the first 12 items on the first 
line, and the remaining items on the second line.

This is neither a bug nor a feature; I believe it is the result of screen layout 
and the browser doing some wrapping.

**Comment by DrSooom** [comment]
@Yegorich555: Open the Browse Mode NVDA Settings and increase the value for "Maximum Number of Characters on One Line".

**Comment by Yegorich555** [comment]
> @Yegorich555: Open the Browse Mode NVDA Settings and increase the value for "Maximum Number of Characters on One Line".

Thank a lot, this option changes behavior. 
Do we have a different way to say to NVDA-reader that content must be readable full at once?

**Comment by Adriani90** [comment]
You could try ctrl+down arrow instead of only down arrow.
It seems this works as expected. Closing as works for me.

**Comment by Adriani90** [comment]
With ctrl+down arrow you navigate paragraph by paragraph and whit includes the whole content of one or more lines which are included in a paragraph.
Another method would be to press i for list item in browse mode. Then NVDA will read item by item, the whole content of every list item.
